docs/apis/streaming/connectors/kinesis.md

```diff
@@ -60,10 +60,10 @@ to setup Kinesis streams. Make sure to create the appropriate IAM policy and use
 
 ### Kinesis Consumer
```

Comment:
I think we should also mention the threading model of the Kinesis consumer in the documentation.

Users should know that each parallel Flink instance will constantly query Kinesis for shards. So if a user has 5 shards, but runs Flink with a parallelism of 50, there will be 50 threads constantly querying Kinesis.

Also, we should explain that there will always be one thread per shard running.